docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -21,6 +21,9 @@ New Features
 * Worker failure detection and high availability failover occurs within 30 seconds. Pulp processes are
   considered missing after 25 seconds, and heartbeats occur every 5 seconds.
 
+* When removing orphan content, the number of items being that have been removed is now stored as
+  the result on the orphan removal task. This count is broken down by content unit type.
```

Comment:
This is also reported by pulp-admin, right? Does it make sense to change this to something like "stored as the result on the orphan removal task and reported by pulp-admin"?